From c721d5877509875c8515df0215fa1dab862013bc Mon Sep 17 00:00:00 2001
From: Darren Tucker <dtucker@dtucker.net>
Date: Fri, 23 Nov 2018 14:11:20 +1100
Subject: [PATCH 127/294] Move RANDOM_SEED_SIZE outside ifdef.

RANDOM_SEED_SIZE is used by both the OpenSSL and non-OpenSSL code
This fixes the build with configureed --without-openssl.
---
 entropy.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/entropy.c b/entropy.c
index 1e3f265..b04ef3f 100644
--- a/entropy.c
+++ b/entropy.c
@@ -24,6 +24,8 @@
 
 #include "includes.h"
 
+#define RANDOM_SEED_SIZE 48
+
 #ifdef WITH_OPENSSL
 
 #include <sys/types.h>
@@ -64,8 +66,6 @@
  */
 #ifndef OPENSSL_PRNG_ONLY
 
-#define RANDOM_SEED_SIZE 48
-
 /*
  * Collect 'len' bytes of entropy into 'buf' from PRNGD/EGD daemon
  * listening either on 'tcp_port', or via Unix domain socket at *
-- 
1.8.3.1

